stylesheets/main.css

```diff
@@ -3256,6 +3256,73 @@ a img {
   color: rgba(0, 0, 0, 0.8);
 }
 
+.content-item-box .product-item-btn {
+  white-space: nowrap;
+  opacity: 0;
+  -webkit-transform: translateY(100%);
+      -ms-transform: translateY(100%);
+          transform: translateY(100%);
+  pointer-events: none;
+  top: 0;
+  -webkit-transition: all 300ms;
+  -o-transition: all 300ms;
+  transition: all 300ms;
+  margin-top: -15px;
+}
+
+.content-item-box .product-item-btn.visible {
+  opacity: 1;
+}
+
+@media screen and (max-width: 640px) {
+  .content-item-box .product-item-btn {
+    margin-top: -12px;
+  }
+}
+
+.content-item-box .product-item-price {
+  -webkit-transform: translateY(0px);
+      -ms-transform: translateY(0px);
+          transform: translateY(0px);
+  pointer-events: all;
+  top: 0;
+  margin: 0;
+  opacity: 1;
+  -webkit-transition: all 300ms;
+  -o-transition: all 300ms;
+  transition: all 300ms;
+  -ms-flex-wrap: wrap;
+      flex-wrap: wrap;
+}
+
+.content-item-box .product-item-details {
+  font-size: 18px;
+  margin-top: 18px;
```

Comment:
Tundub visuaalsel suur vahe.
Praegu:
<img width="745" alt="Screenshot 2021-11-05 at 17 31 49" src="https://user-images.githubusercontent.com/378755/140536807-db97b92d-db83-4039-bdfb-1206b4101d15.png">

Võiks olla pigem:
```suggestion
  font-size: 16px;
  margin-top: 10px;
```

Või
```suggestion
  font-size: 18px;
  margin-top: 8px;
```
<img width="750" alt="Screenshot 2021-11-05 at 17 31 40" src="https://user-images.githubusercontent.com/378755/140536767-79e49a59-645c-470e-9d0d-45f8ac2fb9de.png">